pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -138,6 +145,12 @@ func (n *NodeLabeller) run() error {
 		return err
 	}
 
+	kvmInfoLabels, err := n.getKVMInfoLabels()
```

Comment:
this info executable only needs to run once, right? Is there any reason why it would return different information?

if it only needs to run once, can we execute it once instead of every time the reconcile loop pops?

Reply:
This can return different hyperv results if e.g. host is updated.

Reply:
if the host is updated, won't this involve taking the node offline?

Reply:
I think yes. I updated the code to run kvm plugin only once.